src/main/java/org/prebid/server/handler/AdminHandler.java

```diff
@@ -0,0 +1,80 @@
+package org.prebid.server.handler;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.logging.Logger;
+import io.vertx.ext.web.RoutingContext;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.prebid.server.execution.LogModifier;
+
+import java.util.function.BiConsumer;
+
+public class AdminHandler implements Handler<RoutingContext> {
+
+    private static final String RECORDS_PARAM = "records";
+    private static final String LOGGING_PARAM = "logging";
+
+    private final LogModifier loggerModifier;
+
+    public AdminHandler(LogModifier loggerModifier) {
```

Comment:
Pls rename var `loggerModifier` to `logModifier` or change class name to `LoggerModifier` (and check all project code for var names).